server/service/orbit.go

```diff
@@ -130,6 +130,13 @@ func (svc *Service) EnrollOrbit(ctx context.Context, hostInfo fleet.OrbitHostInf
 
 	secret, err := svc.ds.VerifyEnrollSecret(ctx, enrollSecret)
 	if err != nil {
+		if fleet.IsNotFound(err) {
+			// OK - This can happen if the following sequence of events take place:
+			// 	1. User deletes global/team enroll secret.
+			// 	2. User deletes the host in Fleet.
+			// 	3. Orbit tries to re-enroll using old secret.
```

Comment:
Thanks for adding the comment! Double checking you tested this scenario with Orbit.

Reply:
Yup, the icon is still visible and the logs show the proper error message:

![image](https://user-images.githubusercontent.com/8041318/231872895-d98523b7-282c-4399-8c53-437bef8cfd85.png)

![image](https://user-images.githubusercontent.com/8041318/231872847-63fc5a71-d4cd-49c8-b5c4-b5fd17a513a4.png)